packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ShadowNodeWrapper.kt

```diff
@@ -37,6 +37,10 @@ internal class ShadowNodeWrapper(
             reflectionUtils: ReflectionUtils,
             viewId: Int
         ): ShadowNodeWrapper? {
+            if (!reactContext.hasActiveReactInstance()) {
```

Comment:
seems test for this one is missing

Reply:
We have an issue testing the ShadowNodeWrapper because the ShadowNodes are not mockable